readme.md

```diff
@@ -4,39 +4,37 @@
 
 - List common reasons Javascript is used for server applications
 - Compare and contrast Javascript in the browser vs JS on the server
-- Compare and contrast express.js to Rails / Sinatra.
+- Compare and contrast Express.js to Rails / Sinatra.
 - Use `npm` to manage project dependencies
 - Use `module.exports` and `require` to organize code
-- Use Handlebars templates to simplify rendering in express
+- Use Handlebars templates to simplify rendering in Express
 - Use and configure middleware, e.g. body-parser to handle form submissions
 - Link to static assets in an Express application
 
 ## Opening Framing (15/15)
 
-Let's list out the things we've covered in class...
+Let's start out by listing the things we've covered in the first 3 units of WDI.
 
----
 
-This is SO much stuff. We've pretty much covered the entire stack. Everything we
-cover from here on out is an extension of what we've already learned or a
-different language than we're used to. Today we'll be talking about [expressJS](https://expressjs.com/)
-the "e" in the MEAN stack. Which incidentally is super buzz wordy right now.
+We've moved in and out of several different technological layers with increasing comfort over the course of the class. We've covered the entire stack almost twice over now. We're about to fill in that missing piece by talking about the Javascript back-end.
```

Comment:
Wouldn't say JS back-end is the "missing piece." Your first sentence implied that we've gone through the stack completely multiple times, so really we're presenting an alternate way of doing something we've already done.